src/model/openthaigpt_pretraining_model/models/__init__.py

```diff
@@ -0,0 +1,31 @@
+from .constants import TOKENIZERS, MODELS, MODEL_CONFIGS
+
+
+def load_model(model_config):
```

Comment:
Can we have load_model and load_tokenizer functions for simplicity?

Reply:
No need to refactor config file

Reply:
Done